packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/forms/components/FileUploader/FileUploader.jsx

```diff
@@ -26,6 +26,8 @@ export const FileUploader = ({ messageContent, record, recordFiles }) => {
 
   return (
     <React.Fragment>
+      {/* Hacky solution for form feedback links to be able to scroll to the files uploader */}
+      <label htmlFor="files.enabled"></label>
```

Comment:
Shouldn't this be wrapping around the actual files section text instead of just blank label?

Reply:
@mirekys My bad, I wrapped it around the files accordion label (in nr-docs).